examples/step-59/step-59.cc

```diff
@@ -949,18 +949,15 @@ namespace Step59
 
   template <int dim, int fe_degree>
   LaplaceProblem<dim, fe_degree>::LaplaceProblem()
-    :
 #ifdef DEAL_II_WITH_P4EST
-    triangulation(
-      MPI_COMM_WORLD,
-      Triangulation<dim>::limit_level_difference_at_vertices,
-      parallel::distributed::Triangulation<dim>::construct_multigrid_hierarchy)
-    ,
+    : triangulation(MPI_COMM_WORLD,
+                    Triangulation<dim>::limit_level_difference_at_vertices,
+                    parallel::distributed::Triangulation<
+                      dim>::construct_multigrid_hierarchy)
 #else
     triangulation(Triangulation<dim>::limit_level_difference_at_vertices)
```

Comment:
I think you need to move a copy of the colon here as well.

Reply:
Correct, thank you for noticing (I did not realize this as it is commented out).